src/Layouts/Main.purs

```diff
@@ -64,8 +63,7 @@ component = H.parentComponent
       pure next
 
 
-matchRoutes :: forall eff. H.HalogenIO Query Void (Aff (HA.HalogenEffects eff))
-          -> Eff (HA.HalogenEffects eff) Unit
-matchRoutes app = matches RT.routing (redirects app)
+matchRoutes :: forall eff. H.HalogenIO Query Void Aff -> Effect Unit
+matchRoutes app = match RT.routing (redirects app)
```

Comment:
(redirect app) is underlined red with the error:
```
Could not match type

    t0 -> Routes -> Effect Unit

  with type

    String


while checking that type t0 -> Routes -> Effect Unit
```

Reply:
I understand that `match` is expecting a `String` as the second argument. 
Also I understand thay somewhere in the `query` of `HalogenIO` (here `driver.query`) is the information (that expected String) hidden. But I don't understand "anything" about `launchAff_`.